src/core_modules/capture-core/components/Pages/NewRelationship/RegisterTei/GeneralOutput/WarningsSection/SearchGroupDuplicate/ReviewDialogContentsPager.container.js

```diff
@@ -1,17 +1,28 @@
 // @flow
+import { type ComponentType } from 'react';
 import { connect } from 'react-redux';
-import ReviewDialogContentsPager from './ReviewDialogContentsPager.component';
+import { ReviewDialogContentsPagerComponent } from './ReviewDialogContentsPager.component';
 import { changePage } from './searchGroupDuplicate.actions';
 
+
+export type DispatchersFromFromRedux = {|
+    onChangePage: Function,
+|};
+type PropsFromRedux = {|
+    currentPage: number,
+    nextPageButtonDisabled: boolean,
+|};
+
+export type Props ={| ...DispatchersFromFromRedux, ...PropsFromRedux, ...CssClasses |}
```

Comment:
I would prefer a separate types file, at least when we're exporting the types.

Reply:
Yeah I know we talked about having a separate file :) 

I thought will be _a lot_ of new changes to have it here and since this PR is simply a step for the final work I didnt want to overwhelm you.